src/naive/date.rs

```diff
@@ -232,6 +241,23 @@ fn test_date_bounds() {
 }
 
 impl NaiveDate {
+    /// Returns a `NaiveDate` which corresponds to the current date.
+    #[cfg(all(
+        feature = "clock",
+        not(all(
+            target_arch = "wasm32",
+            feature = "wasmbind",
+            not(any(target_os = "emscripten", target_os = "wasi"))
+        ))
+    ))]
+    pub fn today() -> Self {
```

Comment:
as this is new API surface, potentially it should be fallible?

Reply:
or potentially we don't need this and can just recommend users go `Utc::now().date_naive()` instead

Reply:
Good point, got rid of `NaiveDate::today()` in favor of recommending `Utc::now().date_naive()`.